packages/itwin/grouping-mapping-widget/src/widget/components/GroupPropertyAction.tsx

```diff
@@ -264,90 +97,83 @@ const GroupPropertyAction = ({
   keySet,
   returnFn,
 }: GroupPropertyActionProps) => {
-  const iModelConnection = useActiveIModelConnection() as IModelConnection;
+  const iModelConnection = useActiveIModelConnection();
   const { getAccessToken } = useGroupingMappingApiConfig();
   const mappingClient = useMappingClient();
   const [propertyName, setPropertyName] = useState<string>("");
   const [dataType, setDataType] = useState<DataType>(DataType.Undefined);
   const [quantityType, setQuantityType] = useState<QuantityType>(QuantityType.Undefined);
-  const [classToPropertiesMapping, setClassToPropertiesMapping] =
-    useState<Map<string, Property[]>>();
-  const [ecProperties, setEcProperties] = useState<ECProperty[]>(
+  const [selectedProperties, setSelectedProperties] = useState<PropertyMetaData[]>([]);
+  const [propertiesMetaData, setPropertiesMetaData] = useState<PropertyMetaData[]>(
```

Comment:
Why is this formatted differently than the line above? They are the same length.

Reply:
One line again.